File: December - 09/9 Sum of Unique elements.c
#include <stdio.h>

int main() {
    int n;
    printf("Enter size of array: ");
    scanf("%d", &n);

    int arr[n];
    printf("Enter the elements:\n");
    for(int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    int sum = 0;

    for(int i = 0; i < n; i++) {
        int count = 0;
        for(int j = 0; j < n; j++) {
            if(arr[i] == arr[j]) count++;
        }
        if(count == 1) sum += arr[i]; // appears exactly once
    }

    printf("Sum of unique elements: %d\n", sum);
    return 0;
}

File: December - 09/9.SumofUniqueElements.py
N = int(input("Enter the size of the array: "))
arr = list(map(int, input("Enter the array elements separated by space: ").split()))
freq = {}
for num in arr:
    if num in freq:
        freq[num] += 1
    else:
        freq[num] = 1
unique_sum = 0
for key in freq:
    if freq[key] == 1:
        unique_sum += key
print("Sum of unique elements:", unique_sum)
